react/react-ts-checkout/package.json

```diff
@@ -1,38 +1,29 @@
 {
-    "main": "index.js",
-    "scripts": {
-        "test": "echo \"Error: no test specified\" && exit 1",
-        "start-server": "ts-node server/index.ts",
-        "start-client": "cd client && npm run dev",
-        "dev": "concurrently \"npm run start-server\" \"npm run start-client\""
-    },
-    "keywords": [],
-    "author": "",
-    "license": "ISC",
-    "engines": {
-        "node": "^18.10.0"
-    },
-    "dependencies": {
-        "@types/cors": "^2.8.13",
-        "@types/express": "^4.17.15",
-        "axios": "^0.27.2",
-        "bluebird": "^3.5.0",
-        "concurrently": "^8.0.1",
-        "cors": "^2.8.5",
-        "dotenv": "^16.0.3",
-        "express": "^4.18.1",
-        "help": "^3.0.2",
-        "react-select": "^5.7.0",
-        "request": "^2.81.0",
-        "rewire": "^6.0.0",
-        "tilled-node": "^0.0.4",
-        "ts-node": "^10.9.1"
-    },
-    "devDependencies": {
-        "@types/bluebird": "^3.5.33",
-        "@types/node": "^13.7.7",
-        "@types/request": "^2.48.8",
-        "ts-node": "^10.9.1",
-        "typescript": "~3.8.3"
-    }
```

Comment:
We weren't using a lot of these deps, and request was causing 2 security vulnerabilities. I went ahead and removed the unnecessary ones here, but you might need to do the same in apple pay ex's